DuckDuckGo/LottieView.swift

```diff
@@ -22,35 +22,71 @@ import Lottie
  
 struct LottieView: UIViewRepresentable {
     
+    struct LoopWithIntroTiming {
+        let skipIntro: Bool
+        let introStartFrame: AnimationFrameTime
+        let introEndFrame: AnimationFrameTime
+        let loopStartFrame: AnimationFrameTime
+        let loopEndFrame: AnimationFrameTime
+    }
```

Comment:
Had to get creative with the LottieView so that we could play more involved animations like this one.